src/main/java/seedu/lifeasier/commands/EditNotesCommand.java

```diff
@@ -0,0 +1,145 @@
+package seedu.lifeasier.commands;
+
+import seedu.lifeasier.notes.EmptyNoteListException;
+import seedu.lifeasier.notes.TitleNotFoundException;
+import seedu.lifeasier.storage.FileStorage;
+import seedu.lifeasier.tasks.TaskList;
+import seedu.lifeasier.ui.Ui;
+import seedu.lifeasier.notes.NoteList;
+import seedu.lifeasier.notes.NoteCommandFunctions;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+public class EditNotesCommand extends Command {
+    private static Logger logger = Logger.getLogger(EditNotesCommand.class.getName());
+    private String title;
+
+    public EditNotesCommand(String title) {
+        this.title = title;
+    }
+
+    private void findTitle(Ui ui, NoteList notes, String title) throws TitleNotFoundException {
+        logger.log(Level.INFO, "Start for finding title in note list");
+        int noteNumber = -1;
+        int matchNumber = 0;
+
+        for (int i = 0; i < notes.size(); i++) {
+            if (notes.get(i).getTitle().contains(title)) {
+                matchNumber++;
+                noteNumber = i;
+            }
+        }
+        logger.log(Level.INFO, "End for finding title in note list");
+
+        switch (matchNumber) {
+        case 0:     // no matches
+            logger.log(Level.SEVERE, "No matches for title");
+            throw new TitleNotFoundException();
+        case 1:
+            logger.log(Level.INFO, "One match found");
+            System.out.println(notes.get(noteNumber).toString());
+            ui.showConfirmEditMessage();
+            parseUserResponse(ui, notes, noteNumber, ui.readCommand());
+            break;
+        default:
+            logger.log(Level.INFO, "Multiple matches found");
+            ui.showMultipleMatchesFoundMessage();
+
+            logger.log(Level.INFO, "Start of printing all matching notes");
+            NoteCommandFunctions.printMultipleMatches(notes, title);
+            logger.log(Level.INFO, "End of printing all matching notes");
+
+            noteNumber = Integer.parseInt(ui.readCommand());
```

Comment:
You could add the "-1" logic in this line, so you don't need to keep typing it afterwards

```suggestion
            noteNumber = Integer.parseInt(ui.readCommand()) - 1;
```